tests/console/validate_selinux_policy_sapenablement.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate the presence of the selinux-policy-sapenablement package
+# Maintainer: QE C <qe-c@suse.de>
+
+use strict;
+use warnings;
+use base "consoletest";
+use testapi;
+use version_utils qw(is_sle is_sles4sap);
+
+sub run {
+    die "Test suite designed to run only on SLES4SAP 16.0 and later"
+      unless (is_sle('>=16.0') && get_var('FLAVOR', '') =~ /Minimal-VM-Cloud-sap/);
+
+    validate_script_output(
+        "rpm -q selinux-policy-sapenablement",
```

Comment:
IMO tests should not check for specific package names but the desired effect, i.e. set booleans and modes.